src/main/java/Internship.java

```diff
@@ -0,0 +1,91 @@
+import java.time.YearMonth;
+import java.time.format.DateTimeFormatter;
+
+public class Internship {
+    private String id;
+    private String role;
+    private String company;
+    private YearMonth startDate;
+    private YearMonth endDate;
+    private String skills;
+    private String status;
+
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/yy");
```

Comment:
Can you declare this line first, all static variables need to be declared before instance variables

Refer to "Classes and Interfaces" in this link:
https://se-education.org/guides/conventions/java/index.html